.bash_nav

```diff
@@ -7,3 +7,33 @@ alias desk='cd /mnt/c/Users/gsera/Desktop'
 # source bash
 
 alias srcbash='source ~/.bashrc'
+
+# ---This alias effectively creates a bookmarking app---
+# it adds global symlink shortcuts for use with the 'cds' command
+# ex: 'cds studycloud' will run 'cd -P ~/.short/studycloud'
+# to add a shortcut: go to the target directory and run 'cds + <shortcut_name>' or 'cds + .' to use the current directory name
+# to remove a shortcut: run 'cds - <shortcut_name>' (you can also tack on multiple shortcut names separated by spaces)
+# to list your shortcuts: run 'cds .'
+cds() {
+    if [ ! -d ~/.short ]; then
+        mkdir ~/.short
+    fi
+    if [ $# -eq 1 ] && [ $1 = "." ]; then
+        ls ~/.short/
+    else
```

Comment:
Now, we check what the first parameter is. If it's a `+`, we'll need to create a new shortcut. If it's a `-`, we'll need to remove a shortcut.
Note that we also check that there are the correct number of arguments, otherwise we pass it off to the regular cd command.
Otherwise, we'll want to actually jump to a directory.